rules/react/walmart.js

```diff
@@ -16,15 +16,15 @@ module.exports = {
     // Enforce boolean attributes notation in JSX
     "react/jsx-boolean-value": 2,
     // Validate closing bracket location in JSX
-    "react/jsx-closing-bracket-location": [2, 'tag-aligned'],
+    "react/jsx-closing-bracket-location": [2, "tag-aligned"],
     // Enforce or disallow spaces inside of curly braces in JSX attributes
     "react/jsx-curly-spacing": 0,
     // Enforce event handler naming conventions in JSX
-    "react/jsx-handler-names": 0,
+    "react/jsx-handler-names": 1,
```

Comment:
This one was turned on as a warning. Interested in feedback on whether it should graduate to full error:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md